### ISSUE TITLE
BGP_NEIGHBOR and BGP_MONITORS tables should be unified.

### ISSUE THREAD

**Issue by qiluo-msft** [Azure/sonic-buildimage#9793]
> This table fields seem to be duplicate from the table BGP_NEIGHBOR, please clarify why do we need a separate table

This is a good question, and I believe it is large than the PR trying to solve, ie, add the missing yang model for BGP_MONITORS table, which is existing for long,.

This PR will help minigraph parser unit test improvement, which validate each minigraph generated ConfigDB by SONiC Yang model.

This PR will also unblock the generic config updater end-to-end test.

We can create an issue that BGP_NEIGHBOR and BGP_MONITORS tables should be unified.

_Originally posted by @qiluo-msft in https://github.com/Azure/sonic-buildimage/pull/9752#discussion_r787381518_

**Comment by zhangyanzhao** [comment]
Low priority ask.
